src/websocket/Room.ts

```diff
@@ -1,20 +1,28 @@
 import { ServerWebSocket } from "bun";
+import { Game } from "../logic/Game";
 
 export class Room {
   id: string;
   sockets: ServerWebSocket<unknown>[];
```

Comment:
I forgor to change this type parameter oopsies

Reply:
will wait until changed from unknown